src/sentry/eventstore/snuba/backend.py

```diff
@@ -1,10 +1,51 @@
 from __future__ import absolute_import
 
+from datetime import datetime
 
+from sentry.utils import snuba
+from sentry.models import SnubaEvent
 from sentry.eventstore.base import EventStorage
 
+DEFAULT_START = datetime.utcfromtimestamp(0)  # will be clamped to project retention
+DEFAULT_END = datetime.utcnow()  # will be clamped to project retention
```

Comment:
This will be set once during application boot. Is that what you want?

Reply:
Good call, thanks. That seems wrong in this case.